src/optimizationreporters/OptimizationReporter.C

```diff
@@ -121,3 +122,10 @@ OptimizationReporter::computeObjective()
 
   return val;
 }
+
+void
+OptimizationReporter::setMisfitToSimulatedValues()
```

Comment:
This seems weird.  Maybe the variable is named wrong and should be adjoint parameters instead of misfit parameters.  Then we aren't setting misfit_values to something other than the misfit